content/applications/sales/point_of_sale/pricing/digital_labels.rst

```diff
@@ -0,0 +1,151 @@
+
+.. image:: digital_labels/pricer-module.png
+   :alt: Installing POS Pricer module from Apps
+
+Once the module is activated, configure your :ref:`pricer stores <pricer_tags/stores>` and associate
+:ref:`Pricer tags <pricer_tags/tags>` with your products.
+
+.. _pricer_tags/stores:
+
+Pricer stores
+~~~~~~~~~~~~~
+
+Similarly to the configuration in Pricer, you need to create one pricer store per physical location.
+To do so, go to :menuselection:`Point of Sale --> Configuration --> Pricer Stores`, click
+:guilabel:`New`, and fill in the line with the required information:
+
+- :guilabel:`Store Name`: you can put any name of your liking.
+- :guilabel:`Pricer Tenant Name`: the name of your company account in Pricer, usually followed by
+  `-country_code`. This value is provided by Pricer.
+- :guilabel:`Pricer Login` : the login of your Pricer account.
+- :guilabel:`Pricer Password` : the password of your Pricer account.
+- :guilabel:`Pricer Store ID`: the ID of the related Pricer store as defined on your Pricer
+  database.
+
+.. image:: digital_labels/pricer-stores-setup.png
+   :alt: Configuring a Pricer Store
+
+.. note::
+   - The :guilabel:`Pricer Tags` column is updated automatically when a label is linked to a
+     product.
+   - The :guilabel:`Last Update` and :guilabel:`Last Update Status` columns are updated
+     automatically when the tags are updated.
+
+.. _pricer_tags/tags:
+
+Pricer tags
+~~~~~~~~~~~
+
+For a label to display one specific product information, they both need to be associated with each
+other. To do so:
+
+#. Open the product form by going to :menuselection:`Point of Sale --> Products --> Products` and
+   clicking :guilabel:`New` or selecting an existing product.
+
+   .. warning::
+      If you are creating a new product, configure and save it before associating any Pricer tags.
+
+#. Go to the :guilabel:`Sales` tab, scroll to the :guilabel:`Pricer` section, and select the
+   corresponding :guilabel:`Pricer Store`.
+
+   .. image:: digital_labels/pricer-product.png
+      :alt: Linking Pricer tags to products
+      :scale: 75%
+
+#. Copy the label's ID from the label itself or scan its barcode to fill in the :guilabel:`Pricer
+   tags ids` field.
+
+   .. note::
+      Pricer tag IDs are composed of a letter followed by 16 digits.
+
+.. tip::
+   - We recommended using a barcode scanner to speed up the encoding process.
+   - When setting up Pricer with Odoo for the first time, it is recommended that you configure only
+     one product first. Before configuring more products, ensure you can display its information on
+     a Pricer tag.
+
+Now that you have a product associated with a Pricer tag, we can send its information to Pricer.
+
+Practical application
+---------------------
+
+Odoo automatically sends requests to Pricer to update and synchronize the tags every 12 hours if
+you make any modifications to:
+
+   - A product name, price, barcode, or customer taxes
+   - The currency
```

Comment:
```suggestion
   - Product name, price, barcode, or customer taxes
   - Currency
```
I think we can skip the articles in a bullet list (unless you really want to keep them to differentiate between product specific info in bullet one, and company level currency in bullet two)